native/src/seal/polyarray.cpp

```diff
@@ -123,4 +153,28 @@ namespace seal
 
         is_rns_ = true;
     }
+
+    /**
+    Switches the polynomial array down one modulus by dropping the last modulus
+    in the set.
+    */
+    PolynomialArray PolynomialArray::drop() const {
```

Comment:
Currently this method is not used. I was using it when I was converting the underlying public key ciphertext from a higher modulus to a lower modulus, but the current implementation (using `first_context_data` instead of `key_context_data` already does this drop for us). Leaving this here in case we need it later. I have a test in Rust that ensures it does the correct thing.